api/swagger-spec/v1.json

```diff
@@ -5707,6 +5707,54 @@
       "nickname": "connectGetNamespacedPodExec",
       "parameters": [
        {
+        "type": "boolean",
+        "paramType": "query",
+        "name": "stdin",
+        "description": "redirect the standard input stream of the pod for this call; defaults to false",
```

Comment:
Yes. Seems like those descriptions were not used anywhere before this PR. Will appreciate if you can fix them. Not necessarily required in this PR.